src/distributors/base-installer.ts

```diff
@@ -11,14 +11,17 @@ export abstract class JavaBase {
   protected version: semver.Range;
   protected architecture: string;
   protected javaPackage: string;
+  protected stable: boolean;
 
   constructor(protected distributor: string, installerOptions: JavaInstallerOptions) {
     this.http = new httpm.HttpClient('setup-java', undefined, {
       allowRetries: true,
       maxRetries: 3
     });
 
-    this.version = this.normalizeVersion(installerOptions.version);
+    const javaVersionOptions = this.normalizeVersion(installerOptions.version);
+    this.version = javaVersionOptions.version;
```

Comment:
May be 
```
({version: this.version stable: this.stable} = this.normalizeVersion(installerOptions.version));
```
Or do you think that your approach is more clear?

Reply:
Done. Thank you.